solutions/2018/01_chronal_calibration.rb

```diff
@@ -0,0 +1,24 @@
+class ChronalFrequencies
+  DEFAULT_CYCLE_MAXIMUM = 1000
+
+  def initialize(frequency_change_array)
+    @frequency_change_array = frequency_change_array
+  end
+
+  def sum_all
```

Comment:
look into [Forwardable](https://ruby-doc.org/stdlib-2.5.1/libdoc/forwardable/rdoc/Forwardable.html) for this since it is useless method-wrapping (because it simply calls another method without doing anything else)